gems/activerecord/6.0/activerecord.rbs

```diff
@@ -49,7 +49,6 @@ module ActiveRecord
     def self.create!: (**untyped) -> instance
     def self.validate: (*untyped) -> void
     def self.validates: (*untyped) -> void
-    def self.enum: (Hash[Symbol, untyped]) -> void
```

Comment:
This enum method is different from 6.0 and 7.0. And, this file is shared with other versions.
So, `ActiveRecord::Enum#enum has duplicated definitions` occurred when version 7.0.
Therefore, its line was removed from this file and added signature in each version.